src/pages/index.md

```diff
@@ -1,54 +1,70 @@
+title: Adobe Commerce extensibility
+description: Learn about in-process and out-of-process offerings provided by Adobe Commerce and Magento Open Source.
 keywords:
-  - Extensibility
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
 ---
 
 <Hero slots="image, heading, text"/>
 
 ![Commerce Extensibility](_images/home-bg.jpeg)
 
-# Adobe Commerce Extensibility
+# Adobe Commerce extensibility
 
 Learn how to develop Out-of-Process apps that extend and customize Adobe Commerce.
 
-## Overview
+Extensibility, as a concept, reflects the possibility for growth and the ease of extending functionality in a system. When referring to extensibility in Adobe Commerce, we mean the ability of developers to expand, customize, and control their interactions with Adobe Commerce.
 
-This documentation provides resources for [extending](https://business.adobe.com/products/magento/extended-capabilities.html) and developing Out-of-Process applications for Adobe Commerce.
+The following diagram highlights the main components of the Adobe Commerce [extensibility strategy](https://experienceleague.adobe.com/docs/commerce-operations/implementation-playbook/architecture/extensibility-strategy.html):
 
-### Discover
+![extensibility strategy](./_images/extensibility-strategy-overview.png)
 
-Use these sections to learn about developing Commerce apps.
+Extensibility generally has two variations, in-process extensibility and out-of-process extensibility. In the context of Adobe Commerce, in-process extensibility refers to extended functionality offerings that are located inside or alongside the Adobe Commerce monolith. Conversely, out-of-process extensibility refers to extended functionality and services that operate outside the core software.
 
-<DiscoverBlock slots="link, text"/>
+Out-of-process extensibility can reduce the total cost of ownership by simplifying upgrades and giving developers more control of the timing, coding, and implementation of their systems.
 
-[App Developer Guide](app-development/index.md)
+Other benefits of out-of-process extensibility include:
 
-Learn how to make the transition from developing Commerce PHP extensions to developing Out-of-Process apps.
+- Scalability - Extensions can be scaled separately from the core software, allowing for greater efficiency.
 
-<DiscoverBlock slots="link, text"/>
+- Isolation - An isolated environment means that developers can upgrade or modify their extensions however they want and whenever they want without relying on a core release.
 
-[Events](events/index.md)
+- Technological Independence - Developers can choose whichever tech stacks and coding languages that fit their needs.
 
-Make Commerce transactional data available to App Builder using Adobe I/O. You can define the events to transmit data each time an event triggers, or only under circumstances defined within configuration rules.
+<p></p>
 
-<DiscoverBlock slots="link, text"/>
+## In-process offerings
```

Comment:
I don't like "offerings". Maybe something like "development tools and distribution".  I'm not committed to that phrase by any means.

Also, I think you need to add PHP.

Reply:
@keharper is PHP not covered by Marketplace Extensions?

Reply:
@keharper I guess they can make their own extensions? (not related to the marketplace)